addOns/openapi/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- Content type check in canParseResource(...) is made more specific (Issue 5725).
```

Comment:
These entries should be less code oriented, the user reading this might not know about the code, e.g. `Improve content checks when spidering for specifications (Issue 5725).`

Reply:
Okay.